app/src/main/java/emu/skyline/AppDialog.kt

```diff
@@ -47,6 +66,40 @@ class AppDialog : BottomSheetDialogFragment() {
 
     private val item by lazy { requireArguments().serializable<AppItem>(AppItemTag)!! }
 
+    private val savesFolderRoot by lazy { "${requireContext().getPublicFilesDir().canonicalPath}/switch/nand/user/save/0000000000000000/00000000000000000000000000000001/" }
+    private var lastZipCreated : File? = null
+    private val documentPicker = registerForActivityResult(ActivityResultContracts.OpenDocument()) {
+        it?.let { uri ->
```

Comment:
Just like the export function, I'd also make an `importSave` function here to avoid having all this logic in a variable that isn't strictly related to it. Also add documentation (KDoc) on both the export and import functions.